repos/system_upgrade/common/actors/loaddevicedriverdeprecationdata/actor.py

```diff
@@ -0,0 +1,21 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import ddddload
```

Comment:
Took me some type to decrypt it :) Anyway, it looks like a typo, could we change it?